collections/_article/year-in-review-2024.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "2024 in Review"
+excerpt: "This year was almost as packed as the 4.3 release, so let's go through the months and engine areas together and talk highlights."
+categories: ["newsletter"]
+author: Nat
+image: 
+date: 2024-12-31 10:00:00
+---
+ 
+And with that, the 2024 season comes to an end. What a year it has been, with the number of Godot submissions in gamejams rising to new heights, [more people working for the Godot Foundation than ever before](https://godotengine.org/article/godot-foundation-update-dec-2024/), and most industry award shows featuring one if not more games #MadeWithGodot. 
```

Comment:
```suggestion
And with that, the 2024 season comes to an end. What a year it has been, with the number of Godot submissions in Game Jams rising to new heights, [more people working for the Godot Foundation than ever before](https://godotengine.org/article/godot-foundation-update-dec-2024/), and most industry award shows featuring one if not more games #MadeWithGodot. 
```

Reply:
I think we are falling into the German "capitalize everything" trap - the gap however is correct, I double checked now